movie-mlops-project/model/evaluate.py

```
@@ -0,0 +1,94 @@
+import logging
+import os
+
+import mlflow
+import numpy as np
```

Ruff (0.12.2)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)